package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "vitrine",
+  "version": "1.0.0",
+  "description": "Vous trouverez ici une démonstration d'un panel de mes compétences",
+  "main": "index.js",
+  "directories": {},
+  "scripts": {
+    "dev": "npx nodemon index.js"
```

```suggestion
    "dev": "nodemon index.js"
```

Tu peux te passer du `npx` quand tu es déjà dans un `package.json`